include/lsst/daf/base/DateTime.h

```diff
@@ -59,42 +59,203 @@ class DateTime {
+
+    /**
+     * Construct a DateTime from a double in the specified system and scale
+     *
+     * @param[in] date  specified date
+     * @param[in] system  time system of input (JD, MJD, or EPOCH)
+     * @param[in] scale  time scale of input (TAI, TT or UTC, default TAI).
+     * @throw lsst.pex.exceptions.DomainError if scale is UTC and the date is before 1961-01-01
+     */
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
-    DateTime(int year, int month, int day, int hr, int min, int sec,
-             Timescale scale = TAI);
+
+    /**
+     * Construct a DateTime from year, month, day, etc. (tm struct fields)
+     * 
+     * @param[in] year   year; must be in the range [1902, 2261], inclusive.
+     * @param[in] month  month number, where January = 1
+     * @param[in] day  day of the month (1 to 31).
+     * @param[in] hr  hour (0 to 23)
+     * @param[in] min  minute (0 to 59)
+     * @param[in] sec  integer seconds (0 to 60)
+     * @param[in] scale  time scale of input (TAI, TT or UTC, default TAI).
+     * @throw lsst.pex.exceptions.DomainError if scale is UTC and the date is before 1961-01-01,
+     *      or the year < 1902 or > 2261
+     */
+    DateTime(int year, int month, int day, int hr, int min, int sec, Timescale scale = TAI);
+
+    /**
+     * Construct a DateTime from an ISO8601 string
+     *
+     * Accepts a restricted subset of ISO8601:
+     *     yyyy-mm-ddThh:mm:ss.nnnnnnnnnZ
+     * where:
+     * - the final Z is required for UTC and prohibited for TAI or TT
+     * - the - and : separators are optional
+     * - the decimal point and fractional seconds are optional
+     * - the decimal point may be a comma
+     *
+     * @param[in] iso8601  ISO8601 string representation of date and time
+     * @param[in] scale  time scale of input (TAI, TT or UTC, default TAI).
+     * @throw lsst.pex.exceptions.DomainError if scale is UTC and the date is before 1961-01-01,
+     *      or the year < 1902 or > 2261 or iso8601 is not in an acceptable format
+     */
     explicit DateTime(std::string const& iso8601, Timescale scale);
 
+    /**
+     * Get date as nanoseconds since the unix epoch
+     *
+     * @param[in] scale  desired time scale (TAI, TT or UTC)
+     * @return the date as nanoseconds since the unix epoch in the specified time scale
+     * @throw lsst.pex.exceptions.DomainError if scale is UTC and the UTC date is before 1961-01-01
+     */    
     long long nsecs(Timescale scale = TAI) const;
+
+    /**
+     * Get date as a double in a specified representation, such as MJD
+     *
+     * @return the date in the required system, for the requested scale
+     * @param[in] system  desired time system (JD, MJD, or EPOCH)
+     * @param[in] scale  desired time scale (TAI, TT or UTC)
+     * @throw lsst.pex.exceptions.DomainError if scale is UTC and the UTC date is before 1961-01-01
+     */    
     double get(DateSystem system = MJD, Timescale scale = TAI) const;
```

Comment:
Fixed by making the values not overlap (as you noted in another comment, so I hope this comment simply came earlier and does not indicate dissatisfaction with the fix).